dev/Makefile

```diff
@@ -99,6 +99,18 @@ redeploy-signet-monitoring:
 	$(TF) apply -var="bitcoin_network=signet" \
 				-var="name_prefix=galoy-sig" -auto-approve
 
-
 fmt:
 	$(TF) fmt -recursive
+
+plan:
+	$(TF) plan -sensative -var="bitcoin_network=regtest" -var="name_prefix=galoy-dev"
+
+review:
+	$(TF) show -json | jq
+
+delete:
+	kubectl delete namespace $(ns) --force
```

Comment:
$(ns) is not defined

Reply:
yeah i made a helper because i can never remember the delete command. example usage:

make delete ns=galoy-dev-addons

Reply:
I presume this is similar to the redeploy usecases, so could add instead:

```
redeploy-addons:
	-kubectl delete namespace galoy-dev-addons --force
	cd addons
	$(TF) apply -var="bitcoin_network=regtest" \
				-var="name_prefix=galoy-dev" -auto-approve
```